internal/generator/vector/helpers/helpers.go

```diff
@@ -0,0 +1,30 @@
+package helpers
+
+import (
+	"fmt"
+	"strings"
+)
+
+var Replacer = strings.NewReplacer(" ", "_", "-", "_", ".", "_")
```

Comment:
can reuse `Replacer` from fluentd config by moving to a common package

Reply:
I moved ```Replacer``` from vector to generator helper file now.  I haven't changed fluentd code because ```Replacer``` was getting used at multiple places in fluentd's helper and it looks more readable.